lib/community.js

```diff
@@ -555,6 +555,13 @@ exports.all = {
         width: 150,
         quote: 'hapi allows us to quickly build web-applications, safe in the knowledge that it is feature-complete, battle-tested and production ready. Its plugin system lets us easily break out parts of the system as we scale our dev team. hapi gets out of the way, so we can get on with building business logic.',
         person: 'Antony Jones, Chief Technical Architect'
+    },
+    'Ontoborn': {
```

Comment:
Please put it back in the alphabetical order otherwise LGTM.

Reply:
Done. Please check.